Description: Remove unknown distribution options.
 UserWarning: Unknown distribution option: 'metadata_version'
 UserWarning: Unknown distribution option: 'requires_python'
 UserWarning: Unknown distribution option: 'requires_external'
Author: Bas Couwenberg <sebastic@debian.org>
Forwarded: not-needed

--- a/setup.py
+++ b/setup.py
@@ -307,11 +307,8 @@ extras_require['all'] = list(set(it.chai
 
 setup_args = dict(
     cmdclass={'sdist': sdist_multi_gdal},
-    metadata_version='1.2',
     name='Fiona',
     version=version,
-    requires_python='>=2.6',
-    requires_external='GDAL (>=1.8)',
     description="Fiona reads and writes spatial data files",
     license='BSD',
     keywords='gis vector feature data',
